nengo/_vendor/npconv2d/conv2d.py

```diff
@@ -202,6 +202,39 @@ def conv2d(x, w, pad='SAME', stride=(1, 1)):
     return y
 
 
+def conv2d_groups(x, w, pad='SAME', stride=(1, 1)):
+    """2D convolution (technically speaking, correlation). Compatible with groups > 1
+    Args:
+        x: [N, H, W, C]
+        w: [I, J, C/G, K]
+        pad: [PH, PW]
+        stride: [SH, SW]
+    Returns:
+        y: [N, H', W', K]
+    """
+    ksize = w.shape[:2]
```

Comment:
While I'm thinking of it, we should probably assert that both `x` and `w` have `ndim == 4`, both here and in `conv2d`.